### PR TITLE
Multidb2

### PR DESCRIPTION
**Pull request by mfranco**

### CODE REVIEW

**Comment by mfranco** [comment]
@cormacp @niallryan @Rondineli @vcali  huge refactor

**Comment by mfranco** [comment]
@felipepina01

File: flaskutils/models.py [hunk 1]
```diff
@@ -1,8 +0,0 @@
-from pgsqlutils.orm import BaseModel
-
-
-class FlaskModel(BaseModel):
```
Comment (cormacp) [inline flaskutils/models.py:4]: I suppose this means that our other repos may need to change their imports, if they do something like this : `from flaskutils.models import FlaskModel`?

Comment (mfranco) [reply flaskutils/models.py:4]: Not, only that, all microservices will required an important refactoring but it will happen gradually

**Comment by mfranco** [comment]
:+1:

File: flaskutils/models.py (hunk 1 above)
Comment (cormacp) [reply flaskutils/models.py:4]: Grand, I see. So, if we're working on projects that use `flaskutils` as a dependency, should we wait for this refactoring to take place before rebasing the latest `flaskutils` changes?

Comment (mfranco) [reply flaskutils/models.py:4]: you can use the last release wich corresponds to the last version before this merge

**Pull request merged**